### PR TITLE
[SYCL][ESIMD][EMU] Total tolerated error rate for ESIMD EMU testing i…

### PR DESCRIPTION
**Pull request by lsatanov**
…nvolding floating point computations (due to deviations in HW vs host-based FP computations divergency).

### CODE REVIEW

File: SYCL/ESIMD/mandelbrot/mandelbrot.cpp [hunk 1]
```diff
@@ -164,7 +164,8 @@ int main(int argc, char *argv[]) {
   fclose(dumpfile);
 
   bool passed = true;
-  if (!esimd_test::cmp_binary_files<unsigned char>(out_file, argv[2], 0)) {
+  if (!esimd_test::cmp_binary_files<unsigned char>(out_file, argv[2], 0,
+                                                   0.0027)) {
```
Comment (dongkyunahn-intel) [inline SYCL/ESIMD/mandelbrot/mandelbrot.cpp:168]: Please define tolerance definition macro for `0.0027` along with comment on why percentage-tolerance is added for this test.

Comment (lsatanov) [reply SYCL/ESIMD/mandelbrot/mandelbrot.cpp:168]: BTW, it also should be for ext_intel_esimd_emulator backend only.

Comment (dongkyunahn-intel) [reply SYCL/ESIMD/mandelbrot/mandelbrot.cpp:168]: Check this

https://github.com/intel/llvm-test-suite/blob/857b056e46a44b925f84974a4145d80437ca9cb9/SYCL/ESIMD/matrix_transpose_glb.cpp#L251

Comment (lsatanov) [reply SYCL/ESIMD/mandelbrot/mandelbrot.cpp:168]: Yep, have found it.

**Comment by lsatanov** [comment]
Interestingly, this test specifies async_handler on queue construction but does e.wait() and not e.wait_and_throw()...

**Pull request closed without merging**